### PR TITLE
feat(bigtable): expose internal connection pool (GRPC channel)

### PR DESCRIPTION
**Pull request by vii**

### CODE REVIEW

File: bigtable/bigtable.go [hunk 1]
```diff
@@ -105,6 +105,11 @@ func (c *Client) Close() error {
 	return c.connPool.Close()
 }
 
+// Expose the internal connection pool (GRPC channel) used by the client.
```
[the remark this answers is not in the document]
Comment (crwilcox) [reply bigtable/bigtable.go:108]: @igorbernstein2 @vii is there a reason the approach above wouldn't work?

[the remark this answers is not in the document]
Comment (igorbernstein2) [reply bigtable/bigtable.go:108]: The main reason is to make it easier to keep our internal probers aligned with our official clients. 

Creating a channel pool will require a decent amount of setup duplication:
- endpoint config
- mtls config
- oauth scopes
- useragent
- directpath 
- message sizes
- auth interceptors
- WithBlock

[the remark this answers is not in the document]
Comment (igorbernstein2) [reply bigtable/bigtable.go:108]: Also to give background why we need a reference to the channel pool: we need to enable a couple of experimental features in our internal probers

**Comment by vii** [comment]
thank you so much Cody for showing us a better way to do it behind the scenes!

**Pull request closed without merging**